float32[] angulos
float32[] distancias